TMessagesProj/src/main/java/org/telegram/ui/StickersActivity.java

```diff
@@ -1708,7 +1710,6 @@ protected void onRemoveButtonClick() {
                         StickerSetCell cell = (StickerSetCell) v.getParent();
                         TLRPC.TL_messages_stickerSet stickerSet = cell.getStickersSet();
                         ItemOptions options = ItemOptions.makeOptions(StickersActivity.this, cell);
-                        options.add(R.drawable.msg_archive, LocaleController.getString("StickersHide", R.string.StickersHide), () -> processSelectionOption(MENU_ARCHIVE, stickerSet));
```

Comment:
why delete?

Reply:
There were duplicate menu options so I deleted one

Reply:
removed by [this](https://github.com/NextAlone/Nagram/commit/0e2092d15e5b232445e7dedfd2e596c5c0e7288e)

Reply:
I added it back